integrations/airtable/actions/create-webhook.ts

```diff
@@ -3,30 +3,7 @@ import type { AirtableWebhookCreatedResponse } from '../types';
 import { createWebhookSchema } from '../schema.zod.js';
 
 export default async function runAction(nango: NangoAction, input: CreateWebhook): Promise<WebhookCreated> {
-    const parsedInput = createWebhookSchema.safeParse(input);
```

Comment:
This is replacing too much code. It should only remove the zod specific checks. Not remove the 

```
const { baseId, specification } = parsedInput.data;
    const webhookUrl = await nango.getWebhookURL();

    const config: ProxyConfiguration = {
        // https://airtable.com/developers/web/api/create-a-webhook
        endpoint: `/v0/bases/${baseId}/webhooks`,
        data: {
            notificationUrl: webhookUrl,
            specification
        },
        retries: 10
    };
```

code